tests/supply_chain/test_env_reset.py

```diff
@@ -0,0 +1,503 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import random
+import unittest
+from collections import defaultdict
+from typing import Dict, List
+
+import numpy as np
+
+from maro.simulator.scenarios.supply_chain import ConsumerAction, FacilityBase, ManufactureAction, StorageUnit
+from maro.simulator.scenarios.supply_chain.business_engine import SupplyChainBusinessEngine
+from maro.simulator.scenarios.supply_chain.order import Order
+
+from tests.supply_chain.common import FOOD_1_ID, SKU3_ID, build_env, get_product_dict_from_storage
+
+
+class MyTestCase(unittest.TestCase):
+    """
+    . consumer unit test
```

Comment:
no need to separate the snapshots checking of different units into different tests.

Maybe you can organize the tests to:
- test env reset with None action
- test env reset with ManufactureAction only
- test env reset with ConsumerAction only
- test env reset with both ManufactureAction and ConsumerAction

and also enrich the action onwer.

Reply:
Okay, I will modify it in the direction you said.